students/HarveyHerela/lesson09/mailroom_main.py

```diff
@@ -0,0 +1,110 @@
+import mailroom_db
```

Comment:
You can do `from mailroom_db import Donor, DonorCollection` and then you don't have to type out `mailroom_db` every time you use the classes